src/services/dashboards/dashboard-customize/modules/VariableSelectorDropdown.vue

```diff
@@ -191,33 +193,46 @@ const {
     }
 
     .dropdown-box {
-        @apply flex items-center border border-solid border-gray-300 bg-white rounded-md;
+        @apply flex items-center border border-solid border-gray-300 bg-white rounded-md w-full;
         height: 2rem;
         padding: 0 0.25rem 0 0.75rem;
 
-        .variable-label {
-            @apply text-gray-900 text-label-md;
-        }
+        .variable-contents {
+            @apply inline-flex text-gray-900 text-label-md flex-shrink w-full;
+            max-width: 13.875rem;
 
-        .selected-items {
-            @apply inline-flex items-center;
-            padding-left: 0.5rem;
-
-            .item-for-display {
-                @apply text-gray-900 text-label-md font-bold;
+            .variable-label {
+                @apply flex-shrink-0;
+                max-width: 11rem;
+                overflow: hidden;
+                text-overflow: ellipsis;
+                white-space: nowrap;
             }
-            .selected-count {
-                margin-left: 0.25rem;
+            .item-for-display {
+                @apply font-bold flex-shrink;
+                flex-shrink: 1;
```

Comment:
`flex-shrink`, `flex-shrink: 1; ` entered in duplicate

Reply:
Oh... i missed it.. Thanks!